examples/create_eqpt_sheet.py

```diff
@@ -11,64 +11,72 @@
 determined based on the topology.
 """
 
-from sys import exit
 try:
     from xlrd import open_workbook
 except ModuleNotFoundError:
     exit('Required: `pip install xlrd`')
 from argparse import ArgumentParser
-from collections import namedtuple, defaultdict
 
+PARSER = ArgumentParser()
+PARSER.add_argument('workbook', nargs='?', default='meshTopologyExampleV2.xls',
+                    help='create the mandatory columns in Eqpt sheet')
+ALL_ROWS = lambda sh, start=0: (sh.row(x) for x in range(start, sh.nrows))
 
-Shortlink = namedtuple('Link', 'src dest')
+class Node:
+    """ Node element contains uid, list of connected nodes and eqpt type
+    """
+    def __init__(self, uid, to_node):
```

Comment:
This looks like a good candidate for `@dataclass`, but given that this is not the "core library thing", let's ignore this for now.